tests/end2end/data/downloads/mhtml/complex/complex.html

```diff
@@ -22,7 +22,7 @@
         
         <!-- don't parse non-CSS styles -->
         <style rel="stylesheet" type="text/qss">
-        @import "actually-it's-css";
```

Comment:
Fair enough, it probably just tries to parse and download every stylesheet.

The request-checking should still make sure that our own MHTML loader doesn't try to download it, so it's fine.